app/Services/HandoffService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Controller\Handoff;
+
+class HandoffService
+{
+    public function getAllHandoffsWithControllers() : array
+    {
+        $handoffs = Handoff::all();
+        $handoffArray = [];
+
+        $handoffs->each(function (Handoff $handoff) use (&$handoffArray) {
+            $handoffArray[] = array_merge(
+                $handoff->toArray(),
+                [
+                    'controllers' =>
+                        $handoff->controllers()->orderBy('order')->pluck('controller_position_id')->toArray()
```

Comment:
Is it worth eager-loading the controllers relationship here?

Reply:
Very eager!